azerothcore/azerothcore-wotlk
# Fix(DB/Gossip): show correct gossips for Shadow Priestess Vandis and …

…High Priestess MacDonnell



## Changes Proposed:
-  Add conditions.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/9336
- Closes https://github.com/chromiecraft/chromiecraft/issues/2397

## SOURCE:


## Tests Performed:

- Tested in-game

## How to Test the Changes:


1. .go c id 11055 / .go c id 11053
2. Talk, you shouldn't have any option to talk.
3. .q add 5230 / .q add 5217
4. .q c 5230 / .q c 5217
5. .q rew 5230 / .q rew 5217
6. Talk to her again. You should see that the gossip changed and you have an option to talk to her.
